src/fields/field_constant.impl.hh

```diff
@@ -28,9 +28,9 @@ const Input::Type::Record & FieldConstant<spacedim, Value>::get_input_type(
                                     "Value of the constant field.\n"
                                     "For vector values, you can use scalar value to enter constant vector.\n"
                                     "For square NxN-matrix values, you can use:\n"
-                                    "* vector of size N to enter diagonal matrix\n"
-                                    "* vector of size (N+1)*N/2 to enter symmetric matrix (upper triangle, row by row)\n"
-                                    "* scalar to enter multiple of the unit matrix." )
+                                    " - vector of size N to enter diagonal matrix\n\n"
+                                    " - vector of size (($(N+1)*N/2$)) to enter symmetric matrix (upper triangle, row by row)\n"
+                                    " - scalar to enter multiple of the unit matrix." )
         .allow_auto_conversion("value")
```

Comment:
replace NxN with (($N\timesN$))
replace N with (($N$))
replace (($(N+1)*N/2$)) with (($\frac12N(N+1)$))

... in order to test formulas support in both HTML and Latex little bit.